src/rt-app_parse_config.c

```diff
@@ -785,6 +785,30 @@ static void parse_cpuset_data(struct json_object *obj, cpuset_data_t *data)
 	log_info(PIN "key: cpus %s", data->cpuset_str);
 }
 
+static void parse_numa_data(struct json_object *obj, numaset_data_t *data)
+{
+	data->numaset = NULL;
+	data->numaset_str = get_string_value_from(obj, "nodes", TRUE, NULL);
+
+	if(data->numaset_str != NULL) {
+#if HAVE_LIBNUMA
+		if(numa_available() == -1){
+			log_critical(PIN2 "NUMA is not available");
+			exit(EXIT_INV_CONFIG);
+		}
+#else
+		log_critical(PIN2 "NUMA library is not configured");
+		exit(EXIT_INV_CONFIG);
+#endif
+		data->numaset = numa_parse_nodestring(data->numaset_str);
```

Comment:
Similar question to above, except different point in lifecycle of data.  Do we ever need to free the numaset or numset_str?